spec/helpers/application_helper/toolbar_builder_spec.rb

```diff
@@ -1720,7 +1720,7 @@ def setup_firefox_with_linux
           end
           context "when with snapshots" do
             before { allow(@record).to receive_message_chain(:snapshots, :size).and_return(2) }
-            it_behaves_like 'default case'
+            it_behaves_like 'record with error message', 'remove_snapshot'
```

Comment:
tbh, this spec file is a nightmare. Not only in length, but...

by changing this, you change what is expected as a return value of the subject, which is `build_toolbar_disabled_button`. And it should still be the default case, which is to return false. Yikes. To understand that it took me quite some time. So the problem lies in your refactoring of `supports_control` - See above